src/services/logger.ts

```diff
@@ -1,9 +1,10 @@
 import { createLogger, format, transports } from "winston";
+import type { TransformableInfo } from "logform";
```

Comment:
Did you install `logform`? Can't find it in package.json. Can't also find `winston`.